packages/developer-console-spa/src/components/Sidebar.tsx

```diff
@@ -45,7 +44,7 @@ function Sidebar () {
   ];
 
   function isNavItemActive ( path: string ) {
```

Comment:
Wrap this in a useCallback to avoid unnecessary rendering as it's a function call inside render